features/fixtures/rails4/app/app/jobs/application_job.rb

```diff
@@ -0,0 +1,5 @@
+class ApplicationJob < ActiveJob::Base
+  def perform
```

Comment:
`ApplicationJob` is meant to be an abstract class for other job classes to inherit from, so you should create a separate job class that inherits from `ApplicationJob`.